File: iter3/python_development/Page/SyllabusPage.py
from iter3.python_development.Page.Page import Page
from iter3.python_development.Page.PageType import PageType
from iter3.python_development.System.FunctionType import FunctionType
from iter3.python_development.System.SystemMessage import SystemMessage


class SyllabusPage(Page):

    def __init__(self, content):
        super().__init__(content)
        self.setType("SYLLABUS_PAGE")
        self.setName("Syllabus Page Student")


    def runPage(self):
        self.showContent()
        self.takeInput()
        return SystemMessage("CHANGE_PAGE", "MAIN_MENU_PAGE", None)

File: src/Page/SyllabusPage.java
package Page;

import SystemMessage.FunctionType;
import SystemMessage.SystemMessage;

public class SyllabusPage extends Page{
	
	public SyllabusPage(String content) {
		super(content);
		setType(PageType.SYLLABUS_PAGE);
		setName("Syllabus Page Student");
	}

	@Override
	public SystemMessage runPage() {
		showContent();
		takeInput();
		return new SystemMessage(FunctionType.CHANGE_PAGE, PageType.MAIN_MENU_PAGE, null);
	}
}
